Longest Substring/index.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert');
+
+function getLenOfLongestSubstr(s, k) {
+    assert(s.length > 0, 'String should not be empty');
+    assert(k > 0, 'k should be a positive integer');
+
+    let res = 0;
+    let diff = 0;
+    let i = 0;
+    let j = 0;
+    const store = initStore(s);
+
+    while (j < s.length) {
+        if (diff <= k) {
+            if (store[s[j]] === 0) {
+                diff++;
+            }
+            store[s[j]]++;
+            res = Math.max(res, j - i);
```

Comment:
@dexig recall we were trying to track the len of substring, and it's actually the difference between `j` and `i` 😆

Reply:
lol, it's a good idea 😀